data/index.html

```diff
@@ -185,6 +186,9 @@
         <div style="text-align:center;color:#eaeaea;">
             <h2>Owie</h2>
         </div>
+        <hr>
+        <p style="text-align:center;font-size:11px;">Coefficient: %COEFFICIENT% Offset: %OFFSET%</p>
```

Comment:
You didn't update the placeholders here. This won't work. Please test the code before sending out. Thanks.